### PR TITLE
Refactor aws eks stakater

### PR DESCRIPTION
**Pull request by aliartiza75**

### CODE REVIEW

File: releases/catalog-prometheus-rule.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ metadata:
   labels:
     prometheus: k8s
     role: prometheus-rulefiles
-    release: stakater-monitoring-prometheus-operator
+    release: stakater-monitoring-po
```
Comment (ahmedwaleedmalik) [inline releases/catalog-prometheus-rule.yaml]: This should be `stakater-monitoring-prometheus-operator`, i don't it should be renamed

File: releases/catalog-service-monitor.yaml [hunk 2]
```diff
@@ -7,17 +7,18 @@ items:
   metadata:
     name: stakater-nordmart-catalog-dev-monitor
     labels:
-      release: stakater-monitoring-prometheus-operator
+      release: stakater-monitoring-po
```
Comment (ahmedwaleedmalik) [inline releases/catalog-service-monitor.yaml]: Same here

File: releases/catalog-helm-release.yaml [hunk 3]
```diff
@@ -1,32 +1,50 @@
...
-      imagePullSecrets: "docker-registry-nexus-secret"
+        repository: stakater/stakater-nordmart-catalog
+        tag: v0.0.1
+      fluentdConfigAnnotations:
+        regex: /^(?<time>\\d+(?:-\\d+){2}\\s+\\d+(?::\\d+){2}\\.\\d+)\\s*(?<level>\\S+)
+          (?<pid>\\d+) --- \\[(?<thread>[\\s\\S]*?)\\] (?<class>\\S+)\\s*:\\s*(?<message>[\\s\\S]*?)(?=\\g<time>|\\Z)/
+        regexFirstLine: /^\\d+(?:-\\d+){2}\\s+\\d+(?::\\d+){2}\\.\\d+/
+        timeFormat: "%Y-%m-%d %H:%M:%S.%L"
+      env:
+      - name: MYSQL_USER
```
Comment (ahmedwaleedmalik) [inline releases/catalog-helm-release.yaml:31]: Move user to secret as well

**Review by ahmedwaleedmalik — DISMISSED** [review]

File: README.md [hunk 4]
```diff
@@ -1,3 +1,12 @@
...
 
 A repo to deploy nordmart microservices in dev environment using Flux GitOps
+
+## Configuration Parameters
+
+Before deploying the Nordmart microservice change the following configuration parameters:
+
+| Parameter | Details |
+|---|---|
+| DOMAIN | It will be used to create ingress for the application. It need to be changed in `web` and `gateway` manifest |
```
Comment (ahmedwaleedmalik) [inline README.md]: File references might look better, like `releases/web-helm-release.yaml`

File: releases/catalog-prometheus-rule.yaml (hunk 1 above)
Comment (aliartiza75) [reply releases/catalog-prometheus-rule.yaml]: currently in our monitoring stack Prometheus operator has been named like this and this needs to match that release name. Secondly this was named as `po` intentionally to make the name a little short.

File: releases/catalog-service-monitor.yaml (hunk 2 above)
Comment (aliartiza75) [reply releases/catalog-service-monitor.yaml]: currently in our monitoring stack Prometheus operator has been named like this and this needs to match that release name. Secondly this was named as po intentionally to make the name a little short.

File: releases/catalog-helm-release.yaml (hunk 3 above)
Comment (aliartiza75) [reply releases/catalog-helm-release.yaml:31]: ack

**Review by ahmedwaleedmalik — DISMISSED** [review]

File: releases/docker-cfg-secret.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+data:
```
Comment (usamaahmadkhan) [inline releases/docker-cfg-secret.yaml]: use stringData instead of data

**Review by usamaahmadkhan — APPROVED** [review]

**Pull request merged**